yaksh/test_models.py

```diff
@@ -708,6 +709,9 @@ def setUpClass(self):
         self.trial_course = Course.objects.create_trial_course(self.user)
         self.trial_quiz = Quiz.objects.create_trial_quiz(self.user)
 
+    @classmethod
+    def tearDownClass(self):
+        self.quiz.questionpaper_set.all().delete()
```

Comment:
Deleting trial course should delete everything associated with it. I believe deleting trial course is cleaner. Not really a problem though

Reply:
Ok. Nevermind then!